src/_info.sh

```diff
@@ -30,6 +30,11 @@ function mod_info() {
   info "Disk Statistics"
   msg `check_disk_capacity`
   msg `info_backupDiskUsage`
+  echo >&2 ""
+
+  title "Notifications"
+  info "Summary"
+  msg "`compose_client exec plextracapi npm run status:notifications | fgrep -v '> '`"
```

Comment:
This drops a few extra newlines in the info output. Try chomping the leading un-used lines (`awk` is helpful here).